internal/alb/ls/types.go

```diff
@@ -33,3 +34,16 @@ type ReconcileOptions struct {
 	LoadBalancerArn *string
 	TargetGroups    tg.TargetGroups
 }
+
+const Default404 = "Default 404"
```

Comment:
Will `Default-404` or `default404` be better? since it should be an [serviceName](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names), use 'Default 404' might be broken once k8s core validated the serviceName in ingress spec 😸

Reply:
well since its injected after the ingress has been parsed, i think that is ok. i specifically wanted something that would not be possible for a user to enter into an ingress resource :)

Reply:
got it 😄